app/controllers/users_controller.rb

```diff
@@ -7,15 +7,31 @@ def create
       flash[:notice] = "Account Already Exists"
       redirect_to new_user_path
     else
-      user = User.create(user_params)
-      session[:user_id] = user.id
-      redirect_to root_url
+      user = User.new(user_params)
+      if user.valid?
+        user.save
+        session[:user_id] = user.id
+        redirect_to root_path
```

Comment:
Maybe we want to have the redirect based on where they came from like Krista did in Dinner Dash? Maybe worth opening a :ticket: